src/TriggerBinding/SqlTriggerBindingProvider.cs

```diff
@@ -65,18 +66,26 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
                 return Task.FromResult<ITriggerBinding>(null);
             }
 
-            if (!IsValidTriggerParameterType(parameter.ParameterType))
+            Type parameterType = parameter.ParameterType;
+            if (!IsValidTriggerParameterType(parameterType))
             {
-                throw new InvalidOperationException($"Can't bind SqlTriggerAttribute to type {parameter.ParameterType}." +
-                    " Only IReadOnlyList<SqlChange<T>> is supported, where T is the type of user-defined POCO that" +
-                    " matches the schema of the user table");
+                throw new InvalidOperationException($"Can't bind SqlTriggerAttribute to type {parameter.ParameterType}.");
```

Comment:
I kind of liked the previous one that said very directly what was supported, but I suppose now it'd be a bit too verbose. Some more context would be good at least though I think.

```suggestion
                throw new InvalidOperationException($"Can't bind SqlTriggerAttribute to type {parameter.ParameterType}, this is not a supported type.");
```